### ISSUE TITLE
feature: image as newtab background

### ISSUE THREAD

**Issue by pictureofhotdogplus** [bypassiwastaken/skiovox-helper#75]
it would be cool if you added a feature to set the newtab background to an image
thats all

**Comment by DanTCA1** [comment]
> implemented very soon

Are you using my implementation of it, or are you making your own custom one?

**Comment by bypassiwastaken** [comment]
i honestly dont think this idea is necessary and i dont like it

**Comment by GavinGoGaming** [comment]
> i honestly dont think this idea is necessary and i dont like it

agree. i do see many people having images as their backgrounds but itd be hard to detect light/dark + not super needed atm
